[RM Ch08] Orchestration support for 6G Use cases (#3254)
anuket-project/anuket-specifications: doc/ref_model/chapters/chapter08.rst

  other Cloud Providers;
- network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks.

The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):

- Interoperability: Necessary integration among the orchestration components which would guarantee either the lifecycle management operations or related operation delegation to different orchestration components across the orchestration

> - Flexibiliy: There are use cases which can only be executed if the operations can run disaggregated, otherwise, extreme requirements brough y the use cases could be challenging to achieve taking in account technological constrains (e.g. use cases demanding less than 1 ms of latency)

Reviewer: typo "brough y the use"

Author: It was now corrected in '[2f2e982](https://github.com/cntt-n/CNTT/pull/3254/commits/2f2e9826aabb599ea904427ae80f1557e19b4eeb)'

- not  be the sole proprietary property of a vendor/company
- not be focused on "Transactions/Conversations" or "User/Data Plane" standards (typically IETF, IEEE,
  MEF/Carrier Ethernet etc.)
  

> The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):

Reviewer 2: Please wrap the lines to 120 chars.

> The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):

Author: ```suggestion
The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, 
Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider 
the following attributes (not limited to the below ones):
```

- not  be the sole proprietary property of a vendor/company
- not be focused on "Transactions/Conversations" or "User/Data Plane" standards (typically IETF, IEEE,
  MEF/Carrier Ethernet etc.)
  
The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, 
Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider 
the following attributes (not limited to the below ones):

> - Interoperability: Necessary integration among the orchestration components which would guarantee either the lifecycle management operations or related operation delegation to different orchestration components across the orchestration

Reviewer 3: ```suggestion
- Interoperability: among the orchestrators to support the lifecycle operations or the delegation of operations to other orchestrators
```

> - Flexibiliy: There are use cases which can only be executed if the operations can run disaggregated, otherwise, extreme requirements brough by the use cases could be challenging to achieve taking in account technological constrains (e.g. use cases demanding less than 1 ms of latency)

Reviewer 3: this is a continuation from the text below the bullets
```suggestion
- Flexibility: to support uses that require disaggregated operations; such uses may involve technological constraints (e.g., less than 1 ms of latency)
```

Author: > this is a continuation from the text below the bullets
Please check the following proposal

```suggestion
- Flexibiliy: to support uses that require disaggregated operations; such uses may involve technological constraints brought by the use cases (e.g. less than 1 ms of latency, bandwidth uplink rate at 1 Gbps, 5 nines availability ...)
```

> - Intelligence: AI/ML support either for service lifecycle operation operation or service monitoring, otherwise, the capability of running an unlimited number of services (constrains can be brought by ecosystem/infrastructure variables like processing, memory, storage).

Reviewer 3: Not clear what this is trying to state. Can the approvers please step in and clarify?
